### DISCUSSION TITLE
Publish the jar to artifactory

### DISCUSSION THREAD

**Discussion by mohanraj-nagasamy** [aws/amazon-neptune-jdbc-driver#195] [Q&A]
I can't find the `neptune-jdbc-1.1.0.jar` anywhere.

Is it possible to publish the jar in the following places? 
* Maven Central
* https://aws.oss.sonatype.org

**Comment by lyndonbauto** [comment]
Hey @mohanraj-nagasamy take a look at the bottom of the release for the jar.

https://github.com/aws/amazon-neptune-jdbc-driver/releases/tag/v1.1.0

It currently isn't in maven, sorry.
